src/mitol/google_sheets_refunds/api.py

```diff
@@ -120,3 +120,25 @@ def process_row(self, row_index, row_data):
             result_type=result_type,
             message=message,
         )
+
+    def _int_filter_row(self, row_tuple):
+        """Performs row filtering according to the rules noted in filter_ignored_rows"""
+        try:
+            parsed_data = RefundRequestRow.parse_raw_data(*row_tuple)
+            return not parsed_data.skip_row
+        except Exception:
+            return True
+
+    def filter_ignored_rows(self, enumerated_rows):
+        """
+        Filters skippable rows in the sheet. A row is skippable if these conditions is met:
+        - Ignore flag is set to TRUE
```

Comment:
Could we add here another condition? If the row had been processed already, then it is skippable also.

Reply:
Should now check for either existing errors or a processed date - if either of those are there, it should skip the row.